### PR TITLE
Release 0.26.0

### PR DESCRIPTION
**Pull request by kchibisov**
Starting a new release, if someone aware of what else to include add to a blocked list.

Blocked by:
- [x] #2031
- [x] #2037
- [x] #2047 (#1995)
- [x] #2007
- [x] #2027
- [x] #2053
- [x] #2060
- [x] #2071
- [x] #1957

### CODE REVIEW

**Review by ArturKovacs — APPROVED** [review]
I'm approving for macos even though I'm not a maintainer anymore (but there isn't a macos maintainer right now)

**Comment by maroider** [comment]
The only potentially blocking issue I could find was #1995.

**Comment by tinaun** [comment]
would love to get #2046 in 0.26 if possible, been bothering me for ages

**Comment by filnet** [comment]
I think https://github.com/rust-windowing/winit/pull/2007 could make it into a release.

**Comment by msiglreith** [comment]
@maroider Thanks for the reminder! Opened a PR to bump the version https://github.com/rust-windowing/winit/pull/2047

**Comment by parasyte** [comment]
Nominating #2027 as a blocking issue.

**Comment by maroider** [comment]
I'm hesitant to block on #2046 and #2027 since we don't have a maintainer for macOS ATM.

**Comment by sourcebox** [comment]
#2027 is currently a blocker for all mac apps that uses dialogs and requires either workarounds or using a fork. So merging this PR can only make the situation better. BTW: emilk as the creator of egui is an experienced developer, so there's a good chance it will not break anything.

**Comment by francesca64** [comment]
@sourcebox I'll be reviewing (and merging) both #2027 and #2046.

**Comment by msiglreith** [comment]
Nominating https://github.com/rust-windowing/winit/issues/2060 as this appears to be a regression from 0.25

**Comment by kchibisov** [comment]
Since all linked issues were fixed, we can cut a new release.

**Comment by maroider** [comment]
Can we squeeze in #1957 and #2025?

**Comment by kchibisov** [comment]
I'm not sure about #1957, since it's unclear when the things move. As for the last one I can't say much, depends on web maintainers.

**Comment by adrien-ben** [comment]
I don't know if it's too late but ndk-glue 0.5 was just released. I opened #2071.

**Comment by madsmtm** [comment]
Let's not block on anything further than https://github.com/rust-windowing/winit/pull/1957 and https://github.com/rust-windowing/winit/pull/2025, and for these at a maximum like three or four days.

Anything that makes it in this time-frame goes, everything else will have to wait.

**Review by maroider — APPROVED** [review]
It has been six days now, so I say we push the release through within the next 24 hours or so. #2025 can be included in a patch release.

**Review by francesca64 — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Unreleased
+# 0.26.0 (2021-10-31)
```
Comment (madsmtm) [inline CHANGELOG.md:1]: Remember to update this before the actual release

**Review by madsmtm — APPROVED** [review]

**Pull request merged**